nni/tools/nnictl/updater.py

```diff
@@ -112,11 +112,18 @@ def update_trialnum(args):
     else:
         print_error('Update %s failed!' % 'trialnum')
 
+def load_imported_data(path):
+    '''load the trial data that will be imported'''
+    content = json.dumps(get_json_content(path))
+    if not content:
+        raise ValueError('Imported data should not be empty')
+    return content
+
 def import_data(args):
```

Comment:
Better to have a UT for this.

Reply:
yes, add UT will become a follow-up task after the upcoming release, because @liuzhe-lz said that the time budget for UT is very tight.